kratos/geometries/geometry.h

```diff
@@ -1906,6 +1906,106 @@ class Geometry
         }
     }
 
+    ///@}
+    ///@name Spatial Operations
+    ///@{
+
+    /**
+    * @brief Returns the local coordinates of a given arbitrary point
```

Comment:
The three descriptions are the same

Reply:
The function should do always the same except for the input out parameters